examples/example-gateway/config/production.json

```diff
@@ -27,6 +27,21 @@
 	"clients.corge-http.serviceName": "CorgeHttp",
 	"clients.corge-http.maxConcurrentRequests": 1000,
 	"clients.corge-http.errorPercentThreshold": 20,
+	"clients.corge-http.alternates": {
+	  "routingConfigs": [
+		{
```

Comment:
should we also test with multiple, given that code is trying to do multiple....

Reply:
yeah ill add this